esignet-core/src/test/java/io/mosip/esignet/core/ValidatorTest.java

```diff
@@ -647,4 +682,144 @@ public void test_ClientNameLangValidator_WithInValidDetail_thenFail(){
 		Assert.assertFalse(validator.isValid("abc", null));
 	}
 
+	// =============================ClaimSchemaValidator=============================//
+
+	@Test
+	public void test_ClaimSchemaValidator_withValidDetails_thenPass() throws IOException {
```

Comment:
Why are we deviating from the test method naming convention?

Reply:
following naming of other mehod of this class

Reply:
now chaged to our current naming convention